blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/service/ShardStateIntegrationTest.java

```diff
@@ -38,14 +41,17 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicLong;
 
+import static org.mockito.Mockito.when;
+
 @RunWith(Parameterized.class)
 public class ShardStateIntegrationTest extends IntegrationTestBase {
     
     private ShardStateIO io;
     private List<Integer> shards;
 
-    private final int DELAYED_METRICS_ROLLUP_DELAY_MILLIS = 300000;
+    private final int DELAYED_METRICS_ROLLUP_DELAY_MILLIS = 600000;
     private final int ROLLUP_DELAY_MILLIS = 300000;
+    private final int ROLLUP_WAIT_PERIOD_MILLIS = 500000;
```

Comment:
If you are going to rename the dashboard graphs: short vs long delay, why don't we name the config the same?